### PR TITLE
Initial implementation with Classic parity

### PR DESCRIPTION
**Pull request by ccressent**

### CODE REVIEW

**Review by nikkictl — COMMENTED** [review]
It would be really nice to see some tests for this so we can verify that the metrics payload is what's expected!

**Comment by ccressent** [comment]
> It would be really nice to see some tests for this so we can verify that the metrics payload is what's expected!

Not only nice, but needed! :)
Adding tests today.

File: main.go [hunk 1]
```diff
@@ -2,45 +2,167 @@ package main
...
-			Path:      "example",
-			Env:       "HANDLER_EXAMPLE",
-			Argument:  "example",
-			Shorthand: "e",
+		Host: sensu.PluginConfigOption{
+			Path:      "host",
+			Env:       "SENSU_OPENTSDB_HANDLER_HOST",
+			Argument:  "host",
+			Shorthand: "",
 			Default:   "",
```
Comment (nikkictl) [inline main.go]: Does it not make sense to default to localhost?

**Review by nikkictl — APPROVED** [review]

File: main.go (hunk 1 above)
Comment (ccressent) [reply main.go]: I'm indifferent; I don't think it makes more sense one way or another. If that's what the other plugins default to I can do that too!

Comment (nikkictl) [reply main.go]: I just noticed you had a default port but not a default host. If it's fairly straightforward to set up an opentsdb instance on localhost, and is a common practice then I would say go for it!

**Review by amdprophet — APPROVED** [review]

**Pull request merged**